app/dockerdwrapper.c

```diff
@@ -467,6 +483,14 @@ start_dockerd(const struct settings *settings, struct app_state *app_state)
 
   g_strlcpy(msg, "Starting dockerd", msg_len);
 
+  {
+    g_autofree char *log_level = get_parameter_value(PARAM_DOCKERD_LOG_LEVEL);
+    args_offset += g_snprintf(args + args_offset,
+                              args_len - args_offset,
+                              " --log-level=%s",
```

Comment:
NB! when we switch to rootless, the setting should be on `rootlesskit`. (or maybe both dockerd and rootlesskit). Currently `rootlesskit` only supports `--debug`

Reply:
It will probably be a separate parameter for rootlesskit.